packages/date-time-utilities/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@uifabric/date-time-utilities",
+  "version": "0.1.0",
+  "description": "Date-time utilities",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "test": "just-scripts test",
+    "just": "just-scripts",
+    "code-style": "just-scripts code-style",
+    "clean": "just-scripts clean",
+    "start-test": "just-scripts jest-watch",
+    "lint": "just-scripts lint",
+    "update-api": "just-scripts update-api"
+  },
+  "devDependencies": {
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/es6-promise": "0.0.32",
+    "@types/jest": "^24.9.1",
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@types/webpack-env": "1.15.1",
+    "@uifabric/build": "^7.0.0",
+    "@uifabric/tslint-rules": "^7.2.1",
+    "es6-weak-map": "^2.0.2",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6"
+  },
+  "dependencies": {
+    "@uifabric/set-version": "^7.0.13",
+    "@uifabric/utilities": "^7.20.3",
+    "tslib": "^1.10.0"
+  },
+  "peerDependencies": {
+    "@types/react": ">=16.8.0 <17.0.0",
+    "@types/react-dom": ">=16.8.0 <17.0.0",
+    "react": ">=16.8.0 <17.0.0",
+    "react-dom": ">=16.8.0 <17.0.0"
+  }
```

This package doesn't appear to need react or enzyme dependencies.
```suggestion
  "devDependencies": {
    "@types/jest": "^24.9.1",
    "@types/webpack-env": "1.15.1",
    "@uifabric/build": "^7.0.0",
    "@uifabric/tslint-rules": "^7.2.1"
  },
  "dependencies": {
    "@uifabric/set-version": "^7.0.13",
    "@uifabric/utilities": "^7.20.3",
    "tslib": "^1.10.0"
  }
```